apps/opentelemetry_exporter/rebar.config

```diff
@@ -1,6 +1,6 @@
 {erl_opts, [debug_info]}.
-{deps, [{grpcbox, ">= 0.0.0"},
-        {tls_certificate_check, "~> 1.18"},
+
+{deps, [{grpc, {git, "https://github.com/emqx/grpc-erl", {tag, "0.6.11"}}},
```

Comment:
could you maybe add some words why this change in the PR description?

Reply:
Done